pkgs/build-support/node/build-npm-package/default.nix

```
@@ -37,10 +39,13 @@ let
     hash = npmDepsHash;
   };
 
+  npmWorkspaceFlag = lib.optionalString (!isNull npmWorkspace) "--workspace ${npmWorkspace}";
+  npmWorkspacePrefix = lib.optionalString (!isNull npmWorkspace) "${npmWorkspace}/";
```

We recently removed all calls of isNull
```suggestion
  npmWorkspaceFlag = lib.optionalString (npmWorkspace != null) "--workspace ${npmWorkspace}";
  npmWorkspacePrefix = lib.optionalString (npmWorkspace != null) "${npmWorkspace}/";
```